packages/fx-core/src/plugins/resource/aad/constants.ts

```diff
@@ -49,6 +49,13 @@ export class Constants {
       },
     ],
   };
+
+  static errorInfo = {
```

Comment:
Can we define a class or interface for the error info?

Reply:
This part is for saving constants used in error.ts and aadAppClient.ts.